sale_stock_available_to_promise_release_block/views/sale_order_line.xml

```diff
@@ -54,7 +55,8 @@
     <field name="state">code</field>
     <field name="code">
     if records:
-        records.move_ids.action_block_release()
+      for move in records.move_ids:
+        move.action_block_release()
```

Comment:
Why not changing instead the implementation of `action_block_release` in the parent module instead?
Here https://github.com/OCA/wms/blob/16.0/stock_available_to_promise_release_block/models/stock_move.py#L37-L40
To update with:
```python
    def action_block_release(self):
        """Block the release."""
        for move in self:
            if move.need_release:
                move.release_blocked = True
```

Reply:
It's also an option

Reply:
The PR in the module `stock_available_to_promise` https://github.com/OCA/wms/pull/898